tests/test_reification5.py

```diff
@@ -0,0 +1,74 @@
+#  u(a) :- u(b), k(c) | --output=reify
+# echo "a :- b. k{b}." | clingo --output=reify  
+
+import unittest
+from typing import cast
+from clingo import ast
+from clingox.pprint import pprint
+from clingox.testing.ast import ASTTestCase, parse_statement
+
+from eclingo.config import AppConfig
+from eclingo.parsing import parser
+from eclingo.parsing.transformers import ast_reify, function_transformer
+from tests.test_reification2 import parse_literal
+from clingox.reify import reify_program
+
+def flatten(lst):
+    result = []
+    for lst2 in lst:
+        if isinstance(lst2, list):
+            for e in lst2:
+                result.append(e)
+        else:
+            result.append(lst2)
+
+    return result
+
+def parse_program(stm, parameters=[], name="base"):
+    ret = []
+    parser.parse_program(
+        stm,
+        ret.append,
+        parameters,
+        name,
+        config=AppConfig(semantics="c19-1", verbose=0, use_reification=True),
+    )
+    return flatten(ret)
+
+def clingo_parse_program(stm):
+    ret = []
+    ast.parse_string(stm, ret.append)
+    ret = [rule for rule in ret]
+    return ret
+
+class TestCase(ASTTestCase):
+    def setUp(self):
+        self.print = False
+
+    def assert_equal_program(self, program, expected):
+    
+        # Split and strip whitespaces on '.' based on string given
+        expected_program = [x.strip() for x in expected.split('.') if x]
+        reified_program = [str(sym) for sym in program]
+        
+        # Delete tag
+        del reified_program[0]
```

Comment:
@jorgefandinno  So, if I give a string such as the one:
**atom_tuple(0). atom_tuple(0,1). literal_tuple(0). rule(disjunction(0),normal(0)). atom_tuple(1). literal_tuple(1). literal_tuple(1,1). rule(disjunction(1),normal(1)).**

obtained by using:
**echo "a :- b. k{b}." | clingo --output=reify** 

And I parse it like in the code, and delete the index 0 of the reified program which is the element: tag(incremental)

it will work. Is this the way to go?

Reply:
When I use clingo --output=reify I get the above string,
But when I use reify_program from the library I get an extra term call tag(incremental)

Do you know if there is another -flag to get all of the terms when using the echo command?

Reply:
Can you point me to where you the class Reifier?